pluma/pluma-app.c

```diff
@@ -352,13 +378,23 @@ pluma_app_init (PlumaApp *app)
 
 	/* initial lockdown state */
 	app->priv->lockdown = pluma_settings_get_lockdown (settings);
-}
 
-static void
-app_weak_notify (gpointer data,
-		 GObject *where_the_app_was)
-{
-	gtk_main_quit ();
+	app->priv->extensions = peas_extension_set_new (PEAS_ENGINE (pluma_plugins_engine_get_default ()),
+							PLUMA_TYPE_APP_ACTIVATABLE,
+							"app", app,
+							NULL);
+
+	g_signal_connect (app->priv->extensions,
+ 		  "extension-added",
+			  G_CALLBACK (extension_added),
+			  app);
+
+	g_signal_connect (app->priv->extensions,
+			  "extension-removed",
+			  G_CALLBACK (extension_removed),
+			  app);
```

Comment:
It's customary to write the name of the signal also on the first line. Test:
```
$ git grep g_signal_connect
```

Reply:
done